common/config.go

```diff
@@ -34,4 +34,8 @@ type SourceConfig struct {
 	FetchSize int `json:"fetchSize" default:"50000"`
 }
 
-const DefaultFetchSize = 50000
+const (
+	DefaultFetchSize = 50000
+	// AllTablesWildcard can be used if you'd like to listen to all tables.
+	AllTablesWildcard = "*"
```

Comment:
If tables can be empty, which means "get all tables", why should we also support a special `AllTablesWildcard`? I'd rather support one way of doing things.

Reply:
I honestly think that considering an empty `tables` setting to mean "all tables" is a bit of a footgun. If a user simply forgets to set that field they will be siphoning all the data. On a big database that could create a bit of a headache.

So I'm more in favor of the previous approach, where we consider the setting as required, so the user has to explicitly define `*` if they want all tables.